src/core/Crust/Infobox/Block/HTML/index.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState, useEffect, useRef, useCallback, useLayoutEffect } from "react";
+
+import Icon from "@reearth/components/atoms/Icon";
+import { useT } from "@reearth/i18n";
+import { styled, useTheme } from "@reearth/theme";
+import fonts from "@reearth/theme/fonts";
+
+import { CommonProps as BlockProps } from "..";
+import { Border } from "../utils";
+
+export type Props = BlockProps<Property>;
+
+export type Property = {
+  html?: string;
+  title?: string;
+};
+
+const EventTypes = {
+  mount: "mount",
+} as const;
+
+type MessageEventData = { type: typeof EventTypes.mount; height: number };
+
+const HTMLBlock: React.FC<Props> = ({ block, isSelected, isEditable, onChange, onClick }) => {
+  const t = useT();
+  const theme = useTheme();
+  const { html, title } = block?.property ?? {};
+
+  const ref = useRef<HTMLTextAreaElement>(null);
+  const isDirty = useRef(false);
+  const [editingText, setEditingText] = useState<string | undefined>();
+  const isEditing = typeof editingText === "string";
+  const isTemplate = !html && !title && !isEditing;
+
+  // iframe
+  const [frameRef, setFrameRef] = useState<HTMLIFrameElement | null>(null);
+  const [height, setHeight] = useState(15);
+  const initialScript = `
+window.addEventListener('load', () => {
+  // Initialize styles
+  window.document.body.style.color = "${theme.main.text}";
+  window.document.body.style.margin = "0";
+
+  const rect = window.document.body.getBoundingClientRect();
+  parent.postMessage({
```

Comment:
It should be sent on load and resize event

Reply:
I fixed. [fix: observe resize](https://github.com/reearth/reearth-web/pull/454/commits/6373343d5d88ca05084a4ac114c9c33f10a0c528)